packages/web/server/FirebaseClient.ts

```diff
@@ -10,9 +10,12 @@ async function getFirebase() {
     firebase.initializeApp(publicRuntimeConfig.FIREBASE_CONFIG)
     const loginUsername = publicRuntimeConfig.LOGIN_USERNAME
```

Comment:
what do you think about calling this `FIREBASE_USERNAME`  and same for password, since there could be other logins in the secrets file.

Reply:
Done.